lib/jekyll.rb

```diff
@@ -115,8 +116,8 @@ def configuration(override = Hash.new)
     # timezone - the IANA Time Zone
     #
     # Returns nothing
-    def set_timezone(timezone)
-      ENV['TZ'] = timezone
+    def add_timezone(timezone)
```

Comment:
This is unacceptable, it's an API change, please change this back to what it was. Thanks.